### DISCUSSION TITLE
Supported Python Environments

### DISCUSSION THREAD

**Discussion by chrowe** [microsoft/vscode-jupyter#14349] [Questions and Answers]
When I try to use
```
❯ python --version
Python 3.10.12
```

I get
`The version of Python associated with the selected kernel is no longer supported. Please consider selecting a different kernel.`

https://github.com/microsoft/vscode-jupyter/wiki/Using-unsupported-Python-Environments says "all active versions". Is that the same as "Supported versions" on https://devguide.python.org/versions/#versions ?

**Comment by DonJayamanne** [comment]
I will look into this asap and fix this
